src/types/SessionsType.res

```diff
@@ -1,5 +1,16 @@
 open SdkTypes
 
+type samsungPaySession = {
+  wallet_name: string,
+  version: string,
+  service_id: string,
+  order_number: string,
+  merchant: JSON.t,
+  amount: JSON.t,
+  protocol: string,
+  allowed_brands: array<JSON.t>,
+}
+
```

Comment:
Are we getting a new object from backend for samsung pay or we can merge with sessions types?
As @AashuJuspay has all session objects in one type.

Reply:
As of current implementation, we have a single session type for session tokens of google pay and apply pay, so added some samsung pay specific fields here